thriftserver/thriftserver/base/thriftserver-db-service.yaml

```diff
@@ -0,0 +1,21 @@
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: thriftserver-db-service
+  annotations:
+    template.openshift.io/expose-uri: |
+      postgres://{.spec.clusterIP}:{.spec.ports[?(.name=="postgresql")].port}
+spec:
+  ports:
+    - name: postgres
+      protocol: TCP
+      port: 5432
+      targetPort: 5432
+      nodePort: 0
```

Comment:
Can we remove `nodePort`?

Reply:
I don't see a reason to keep it. Removed.